reference/network-ports.md

```diff
@@ -31,7 +31,6 @@ For the Anbox Cloud Appliance, ports are exposed only for accessing the Anbox Cl
 
 | Service               | Port(s)     | Protocol  | Exposed externally | Required | Description                            |
 |-----------------------|-------------|-----------|--------------------|----------|----------------------------------------|
-| AMS node controller   | 10000-11000 | UDP & TCP | yes                | no       | Instance service ports                 |
```

Comment:
We can't just remove it. The ports can still be exposed but are for the appliance now on LXD itself. So this needs to just change "AMS node controller" to "LXD" and not drop the entire line.